pkg/services/authn/authnimpl/service.go

```diff
@@ -320,6 +324,10 @@ func (s *Service) RegisterClient(c authn.Client) {
 	}
 }
 
+func (s *Service) GetClient(name string) authn.Client {
+	return s.clients[name]
+}
```

Comment:
This can be nil, so either we go with (authn.Client, bool) for this function or we change to the other one we discussed `IsClientEnabled` and handle this internally

Reply:
@kalleep I followed your suggestions and have implemented an `IsClientEnabled` that handles the client logic